src/geometry/warp.jl

```diff
@@ -32,18 +32,21 @@ function Homography{T}() where T
     return Homography{T}(SMatrix{3, 3, T, 9}(m))
 end
 
-function (h::Homography{T})(x::SVector{3,K}) where {T,K}
-    y = h.H * x
-    return SVector{2,T}(y[1:2] ./ y[end])
+function (h::Homography{K})(x::SVector{3, L}) where {L, K}
```

Comment:
`V` can be also removed from this and some other cases:

```suggestion
function (h::Homography{K})(x::SVector{3}) where {K}
```

Reply:
Yes, this does the same thing. I'll change it as you say.